crowbar_framework/app/controllers/barclamp_controller.rb

```diff
@@ -40,6 +40,39 @@ def controller_to_barclamp
 
   add_help(:barclamp_index)
   api :GET, "/crowbar", "Returns a list of string names and descriptions for all barclamps"
+  example '
```

Comment:
I'd assume barclamp is an attribute of the crowbar entity, different than Version, for instance